pkg/local/server_generic.go

```diff
@@ -3,11 +3,9 @@
 package local
 
 import (
-	"context"
-	_ "embed"
 	"fmt"
 )
 
-func ServerRun(ctx context.Context) error {
+func ServerRun() error {
```

Comment:
@thedadams the signature for this function in `pkg/local/server_linux.go` needs to be changed as well. I think that explains why this is failing in CI, but not locally.

Reply:
Thanks for catching that.